content/en/docs/external-add-ons/feast/introduction.md

```diff
@@ -0,0 +1,99 @@
+- __Standardized definitions__: Feast becomes the single source of truth for all feature definitions and data within an organization.
+  Teams are able to capture documentation, metadata, and metrics about features. 
+  This allows teams to communicate clearly about features, test feature data, and determine if a feature is both safe and relevant to their use cases.
+
+- __Historical serving__: Features that are persisted in Feast can be retrieved through its feature serving APIs to produce training datasets. 
+  Feast is able to produce massive training datasets that are agnostics of the data source that was used to ingest the data originally. 
+  Feast is also able to ensure point-in-time correctness when joining these data sources, which in turn ensures the quality and consistency of features reaching models.
+
+- __Online serving__: Feast exposes low latency serving APIs for all data that has been ingested into the system. 
+  This allows all production ML systems to use Feast as the primary data source when looking up real-time features.
+
+- __Consistency between training and serving__: Feast provides a consistent view of feature data through the use of a unified ingestion layer, unified serving API and canonical feature references. 
+  By building ML systems on feature references, teams abstract away the underlying data infrastructure and make it possible to safely move models between training and serving without a drop in data consistency.
+
+- __Feature sharing and reuse__: Feast provides a discovery and metadata API that allows teams to track, share, and reuse features across projects. 
+  Feast also decouples the process of creating features from the process of consumption, meaning teams that start new projects can begin by simply consuming features that already exist in the store, instead of starting from scratch.
+
+- __Statistics and validation__: Feast allows for the generation of statistics based on features within the systems.
+  Feast has compatibility with TFDV, meaning statistics that are generated by Feast can be validated using TFDV.
+  Feast also allows teams to capture TFDV schemas as part of feature definitions, allowing domain experts to define data properties that can be used for validating these features in other production settings like training, ingestion, or serving.
+
+### What is a feature store?
+
+Feature stores are systems that reduce challenges faced by ML teams when productionizing features.
+
+Some of the key challenges that feature stores help to address include:
+
+- __Feature sharing and reuse__: Engineering features is one of the most time-consuming activities in building an end-to-end ML system, yet many teams continue to develop features in silos. 
+  This leads to a high amount of re-development and duplication of work across teams and projects.
+
+- __Serving features at scale__: Models need data that can come from a variety of sources, including event streams, data lakes, warehouses, or notebooks. 
```

Comment:
```suggestion
- __Serving features at scale__: Models need data that can come from a variety of sources, including api-calls, event streams, data lakes, warehouses, or notebooks. 
```

Reply:
Done in https://github.com/kubeflow/website/pull/3836/commits/a5e72b551cea20a364a90aed4f6ef4f558a2582e